packages/matchbox/src/components/Grid/Grid.js

```diff
@@ -1,18 +1,29 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
-import { pick } from '../../helpers/propTypes';
```

Comment:
I think this is the only place using this function.  It should probably be deleted.